scenes/game/card_base.gd

```diff
@@ -34,6 +34,8 @@ const CardBaseScene = preload("res://scenes/game/card_base.tscn")
 @onready var active_skill_name = $OuterMargin/ActiveSkillIndicator/PanelContainer/HBoxContainer/SkillContainer/ActiveSkillName
 @onready var targeted_damage_indicator = $OuterMargin/TargetedDamageIndicator
 @onready var targeted_down_indicator = $OuterMargin/TargetedDownIndicator
+@onready var no_proxy_panel = $PanelContainer
```

Comment:
I'm not a huge fan of "no_proxy" as a descriptive name for what this is.
Maybe popout_info_panel or something?